packages/semver/src/executors/version/utils/get-last-version.spec.ts

```diff
@@ -31,16 +31,20 @@ describe(getLastVersion.name, () => {
     expect(tag).toEqual('2.1.0');
   });
 
-  it('should compute current version from previous semver prerelease tag', async () => {
+  it('should compute current version from previous semver prerelease tag with corresponding preid', async () => {
     mockGitSemverTags.mockResolvedValue([
-      'my-lib-2.1.0-beta.0',
+      'my-lib-2.1.0-beta.5',
+      'my-lib-2.1.0-alpha.0',
```

Comment:
Is it possible to unit test custom pre ids as well like `add-feature` and `fix-bug`.

Reply:
sure, no prob